ext/strings_test.go

```diff
@@ -338,256 +338,296 @@ func TestStringsWithExtension(t *testing.T) {
 
 func TestStringFormat(t *testing.T) {
 	tests := []struct {
-		name           string
-		format         string
-		dynArgs        map[string]any
-		formatArgs     string
-		locale         string
-		err            string
-		expectedOutput string
+		name               string
+		format             string
+		dynArgs            map[string]any
+		formatArgs         string
+		locale             string
+		err                string
+		expectedOutput     string
+		enableCompileCheck bool
```

Comment:
nit: If almost all tests should do a compile check, invert this boolean (e.g. `skipCompileCheck`) so it doesn't need to be set repeatedly on almost all the tests?

Reply:
Makes sense, I've flipped the bool and renamed the field to `skipCompileCheck`.